grease-cli/tests/Shape.hs

```diff
@@ -229,7 +239,7 @@ ptrShape ::
   [PtrShape.MemShape 64 NoTag] ->
   Shape LLVM NoTag (LLVMPointerType 64)
 ptrShape offset =
-  Shape.ShapeExt . PtrShape.ShapePtr NoTag offset . PtrShape.PtrTarget . Seq.fromList
+  Shape.ShapeExt . PtrShape.ShapePtr NoTag offset . flip PtrShape.PtrTarget Nothing . Seq.fromList
```

Comment:
Seems like this constructor gets `flip`ed a lot. Should we reverse the arguments?